### PR TITLE
Update grpcbox for chatterbox performance work

### PR DESCRIPTION
**Pull request by Vagabond**
Depends on tsloughter/chatterbox#22

### CODE REVIEW

**Comment by tsloughter** [comment]
I think the github actions Ci needs to be updated to ubuntu 20.04 or it won't run.

**Comment by tsloughter** [comment]
Yea, it is bitrot. If you have any ideas though let me know :). I've been banging my head against chatterbox on otp-25/26.

Tests are going to fail on 23 so need to drop that.

**Comment by tsloughter** [comment]
Ah yup, I thought I tried adding that back to allowed and it still failed so thought it wasn't the issue. But generating new keys worked.

**Comment by Vagabond** [comment]
Ok, I will rebase once you have new keys merged.

File: rebar.config
Comment (tsloughter) [inline rebar.config:3]: Switch this to `tsloughter/chatterbox` and I'll merge after the rebase. Not ready to make a chatterbox release with such big changes yet but want it easy to test it for others.

**Review by tsloughter — APPROVED** [review]

**Pull request merged**
